### DISCUSSION TITLE
Verified domain still require additional proof

### DISCUSSION THREAD

**Discussion by SaPinYshMaZe** [community/community#57507] [GitHub Education]
### Hi everyone,

Hi,

Our school is a verified as a Global Campus and we've also verified our domain to ease the sign up process for our students but whenever a student signs up using their school issued email which is from our verified domain, they still get the message to provide additional proof. Our students don't have any student ID or easy access to officially dated documents which leads to basically all of them just not signing up.

What can we as a school do in order for our students to no longer be required to provide additional proof?

Addendum:
Our students are signing up from our campus at our stated address so geo-verification should also not be a problem.

Here are pictures to clarify. Domain is verifed and correct, students have correct emails.
![MicrosoftTeams-image (1)](https://github.com/community/community/assets/62467504/cf3c80bd-83d2-43ac-9bc5-c7c7a929391c)

What can we as a school do so that this next step is not required?
![MicrosoftTeams-image](https://github.com/community/community/assets/62467504/a88ffb47-0639-4ed5-88a7-f77612d17969)

**Comment by chriskthomas** [comment]
I think they require more verification because of the prevalence of alumni email addresses. I know that when you sign up for campus, you can indicate that your email is not available to alumni, but I'm not sure if they use this.

**Reply by SaPinYshMaZe** [reply]
Sure that might be true, but my question is what can we as the organisation/school do so as to remove this requirement? We know that other schools don't require this additional verification. We would much rather put in the legwork on our side rather then putting it on our students.

**Comment by samus-aran** [comment]
Hi @SaPinYshMaZe 

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by SaPinYshMaZe** [reply]
As far as I can tell, all of our schools details are correct. Address is correct, domain is correct and verified. Our students go through the Student Application form and select their school issued email. The problem is that they then get a message requiring them to submit additional proof of academic status. Since they don't have any school ID or other easy proof to submit they students give up at that point. For those student who acquire a signed and stamped letter stating that they are students it doesn't seem to be a problem at all. We just need help avoiding that additional step...

We submitted a ticket to support before posting this but the only answer we got was to turn to GitHub Community.
